fabric-example/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "fabricexample",
+  "version": "0.0.1",
+  "private": true,
+  "scripts": {
+    "android": "react-native run-android",
+    "build:android": "mkdirp dist/res && react-native bundle --entry-file index.js --platform android --dev true --bundle-output dist/main.android.jsbundle --assets-dest dist/res",
+    "build:ios": "mkdirp dist && react-native bundle --entry-file index.js --platform ios --dev true --bundle-output dist/main.ios.jsbundle --assets-dest dist",
+    "build:macos": "mkdirp dist && react-native bundle --entry-file index.js --platform macos --dev true --bundle-output dist/main.macos.jsbundle --assets-dest dist",
+    "build:windows": "mkdirp dist && react-native bundle --entry-file index.js --platform windows --dev true --bundle-output dist/main.windows.bundle --assets-dest dist",
+    "ios": "react-native run-ios",
+    "lint": "eslint .",
+    "macos": "react-native run-macos --scheme example",
+    "start": "react-native start",
+    "test": "jest",
+    "windows": "react-native run-windows --sln windows/example.sln"
+  },
+  "dependencies": {
+    "react": "18.1.0",
+    "react-native": "0.70.1",
+    "@react-native-community/progress-view": "link:../"
```

Comment:
Using link: protocol, will need to update `metro.config.js` to watch the whole repo and blacklist react-native but it will make `yarn install` faster.